cfme/test_framework/browser_isolation.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+"""Plugin enabling us to isolate browser sessions per test."""
+import pytest
+
+
+def pytest_addoption(parser):
```

Comment:
I'm not sure what it means to isolate a browser session.   Perhaps you could add a comment explaining what you mean by that.

Reply:
Sure I can. Basically it is ensuring that each test will start a fresh browser when needed. If we have a browser screwup again, this is one of the things that may help reducing the failure propagation further (like what happened recently).

Reply:
Sweet, makes sense.